vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -301,6 +368,256 @@ public void test_updateContentOrder_forNonExistentId() throws BlockDoesNotExistE
     }
 
     @Test
+    public void test_getAllContentBlocks_success() throws BlockDoesNotExistException {
+
+        String slideId = "slide1";
+        ISlide slide = new Slide();
+        slide.setId(slideId);
+        String contentBlockIdString = "contentBlockId1";
+        ContentBlock firstContentBlock = new ContentBlock();
+        firstContentBlock.setId(contentBlockIdString);
+        firstContentBlock.setContentOrder(Integer.valueOf(3));
+
+        List<IContentBlock> contentBlocks = new ArrayList<>();
+        contentBlocks.add(firstContentBlock);
+        slide.setContents(contentBlocks);
+        when(slideManager.getSlide(slideId)).thenReturn(slide);
+
+        List<IContentBlock> returnedContentBlock = managerToTest.getAllContentBlocks(slideId);
+        assertEquals(returnedContentBlock.get(0).getId(), contentBlockIdString);
+        assertEquals(returnedContentBlock.size(), contentBlocks.size());
+        assertEquals(returnedContentBlock.get(0).getContentOrder(), Integer.valueOf(3));
+        assertTrue(!returnedContentBlock.isEmpty());
```

Comment:
if the above asserts are true, then this one will be too (or if this one returns false, then line 387 would have already thrown an exception)